src/util/normalize-manifest/fix.js

```diff
@@ -185,11 +188,17 @@ export default async function (
     if (!info.bin && binDir && typeof binDir === 'string') {
       const bin = info.bin = {};
 
-      for (const scriptName of await fs.readdir(path.join(moduleLoc, binDir))) {
-        if (scriptName[0] === '.') {
-          continue;
+      try {
```

Comment:
fs.exists is deprecated.  I'm not sure how long it has been deprecated and the other options of fs.stat() or fs.access() seem like they would have the same problem of having the error thrown.